css/styles.css

```diff
@@ -0,0 +1,301 @@
+html, body{
+    background-color: #FFFFFF;
+    font-family: Calibri, Tahoma, Geneva, Verdana, sans-serif;
+}
+
+header{
+    background-color: aqua;
+    position: absolute;
+    width: 1920px;
+    height: 840px;
+    left: 0px;
+    top: 0px;
+}
+
+.JK{
```

Comment:
Keep all classnames in lowercase. Do not skip whitespaces between classname and open curly brace. Fix this issues in the layout at whole.